experiments/standalone/Bucket/compare_gpu_cpu_output.jl

```diff
@@ -4,6 +4,7 @@ import ClimaLand
 outdir = joinpath(pkgdir(ClimaLand), "experiments/standalone/Bucket/artifacts")
 cpu_state = readdlm(joinpath(outdir, "tf_state_cpu.txt"), ',')
 gpu_state = readdlm(joinpath(outdir, "tf_state_gpu.txt"), ',')
-@show mean(cpu_state .- gpu_state)
-@show eps(Float64)
-@assert mean(cpu_state .- gpu_state) < eps(Float64) * 5e2
+@show abs(maximum(cpu_state .- gpu_state))
+@show abs(median(cpu_state .- gpu_state))
+@show abs(mean(cpu_state .- gpu_state))
+@assert isapprox(cpu_state, gpu_state)
```

Comment:
@juliasloan25 what do you think of this?

Instead of comparing only one value, we compare the entire state.

Reply:
That looks good! Is there a way to set the tolerance for `isapprox` in case they're too different?

Reply:
Yes, you can set `atol` and `rtol` if you want to